create_aruco_dict.py

```diff
@@ -0,0 +1,25 @@
+import sys
+sys.path.append('/usr/local/lib/python2.7/site-packages')
+import cv2
+import numpy as np
+import cv2.aruco as aruco
+
+#small_dict = aruco.generateCustomDictionary(6, 3)
+NUM_OF_MARKERS = 6
+small_dict = aruco.Dictionary_create(NUM_OF_MARKERS, 3)
```

Comment:
I'm confused by this - is this supposed to be run as a standalone script? Should it not be defined as a function, with the multi line comment acting as a doc string?

Reply:
Sorry this isn't clearer. This is a standalone script to generate the image files for the Aruco markers we're using, so only needs to be run once.  that docstring should be converted into a comment, I think its purpose at the time was to  remind me of the arguments of drawMarker.